app/views/policies/_nominies_details.html.slim

```diff
@@ -0,0 +1,56 @@
+form.form-horizontal.form-label-left
+  .col-md-6
+    .form-group
+      label.control-label.col-md-3 for="first-name" 
```

Comment:
@KaranBamaniya-BTC : Use the same partial for personal fields as we are having the same fields in sign up as well